pkg/providers/openshift/provider_postgres.go

```diff
@@ -35,7 +35,7 @@ var (
 	// default openshift create paramaters
 	defaultPostgresPort        = 5432
 	defaultPostgresUser        = "postgresuser"
-	defaultPostgresPassword    = "password"
+	defaultPostgresPassword, _ = resources.GeneratePassword()
```

Comment:
Do we need this here? We generate the password and handle the error when creating the [secret](https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/providers/openshift/provider_postgres.go#L108). We only use this `defaultPostgresPassword` if the `password` is nil after we generate it.

Reply:
Only spotted it here https://github.com/integr8ly/cloud-resource-operator/blob/6c7aaf3c5c6dd75fe33d913a59a7763c5a848e90/pkg/providers/openshift/provider_postgres.go#L505 but I see what you mean https://github.com/integr8ly/cloud-resource-operator/blob/6c7aaf3c5c6dd75fe33d913a59a7763c5a848e90/pkg/providers/openshift/provider_postgres.go#L108

Agree, will close this pr as no value.